cosinnus_conference/utils.py

```diff
@@ -55,8 +57,14 @@ def render_template(dynamic_fields, field_name, field_type, user, group):
     if not recipients:
         recipients = group.actual_members
     
+    # check if conference has the `Request application` method: 
+    # only in this case the email notification is allowed to ignore the `never` notification setting
     for recipient in recipients:
-        if not check_user_can_receive_emails(recipient, ignore_user_notification_settings=True):
+        ignore = False
+        if CosinnusConference.objects.filter(membership_mode=CosinnusConference.MEMBERSHIP_MODE_APPLICATION) \
+            and CosinnusConferenceApplication.objects.filter(Q(conference=group) & Q(may_be_contacted=True) & Q(user=recipient)):
```

Comment:
No need to chain Q(...) statements inside a single filter(...) expression to AND them.
`filter(Q(conference=group, may_be_contacted=True, user=recipient))` would be equal and better readable.